frontend/components/shelter/pet/add/FormComponents.tsx

```diff
@@ -300,9 +300,9 @@ const DataField = ({
 };
 
 interface ImageGalleryProps {
-	images?: string[];
-	onChange?: (images: string[]) => void;
-	isEditMode?: boolean;
+	images: string[];
```

Comment:
I checked references to this `ImageGallery` and they all have props that will have these props.